### ISSUE TITLE
Crash when starting a new game in Obscure: the aftermath.

### ISSUE THREAD

**Issue by blackman91** [hrydgard/ppsspp#9114]
Using the latest dev build on android the game crashes after the initial cutscene, this does not happen in the stable 1.3 build.

**Comment by Zetchzie** [comment]
works fine on windows 10 x64

**Comment by blackman91** [comment]
That is why I said on Android....

**Comment by hrydgard** [comment]
Might have something to do with the recent build flag changes. I'll take a look as soon as I get a chance.

**Comment by sum2012** [comment]
@blackman91 what existly git version did you test ?

**Comment by moura464** [comment]
Build  v1.3g1234abcd  v1.4g1234abcd version :) !!!

**Comment by blackman91** [comment]
I used the latest: https://github.com/hrydgard/ppsspp/commit/9c08b60564c0071e74a429982b9ad14935790e05

Also the same happens in ys ark of napishtim, crashes when loading an ingame save or starting a new game.
They work fine in the 1.3 stable.

**Comment by mrcmunir** [comment]
The crash is related when vertex cache is enabled.
Disable Vertex cache the game is working fine in my phone.

**Comment by sum2012** [comment]
@blackman91 Did you mean before this screen crash ?
![screenshot_20161108-215124](https://cloud.githubusercontent.com/assets/2177532/20101622/c8467e14-a5fd-11e6-8d6b-a04cda220423.jpg)

**Comment by blackman91** [comment]
Yes it crashes right after the intro cutscene, I tested in two devices with a mali gpu different android os 4.4 and 4.2.2

**Comment by mrcmunir** [comment]
@blackman91 disable vertex cache option will help the crash in my case working if disable that option.

**Comment by hrydgard** [comment]
Thanks guys, this may be related to the very similar iOS crash.

I'll look into it soon.

**Comment by sum2012** [comment]
Asus zenfene 3 doesn't crash
Samsung Galaxy Tab S 8.4 crash

**Comment by sum2012** [comment]
Many people in baidu forum report many game crash on neweset version,maybe related to this

**Comment by sum2012** [comment]
@hrydgard 
good
v1.3-136-g6e7e62f
bad
v1.3-143-gea5e9f8

compare:  https://github.com/hrydgard/ppsspp/compare/v1.3-136-g6e7e62f...v1.3-143-gea5e9f8

**Comment by Nucleoprotein** [comment]
I think this related to run-time NEON detection on ARM v7 (which seems to be disabled by define before v1.3-143-gea5e9f8). I think that change is only one that can cause crashes.

**Comment by hrydgard** [comment]
Got a repro on this, it's trying to hash unaligned data, which blows up with NEON but not with ARM. I'll fix.

**Comment by Orphis** [comment]
Good find!
